clippy_lints/src/dbg_macro.rs

```diff
@@ -15,6 +15,13 @@ declare_clippy_lint! {
     /// `dbg!` macro is intended as a debugging tool. It
     /// should not be in version control.
     ///
+    /// ### Known problems!
+    /// * The lint level can't be changed by attributes, like most other lints.
```

Comment:
It can, just not by crate-level inner attributes. Inner-attributes on modules or other item-likes work, as well as all outer-attributes.

Reply:
Ohh, I misunderstood the issue discussion, thanks for pointing that out :upside_down_face:

Reply:
Updated the comment, I hope this time it's correct :upside_down_face: